tests/modules/seq2seq_encoders/pytorch_seq2seq_wrapper_test.py

```diff
@@ -124,6 +124,46 @@ def test_wrapper_raises_if_batch_first_is_false(self):
             lstm = LSTM(bidirectional=True, num_layers=3, input_size=3, hidden_size=7)
             _ = PytorchSeq2SeqWrapper(lstm)
 
+    def test_wrapper_works_when_passed_state_with_zero_length_sequences(self):
+        lstm = LSTM(bidirectional=True, num_layers=3, input_size=3, hidden_size=7, batch_first=True)
+        encoder = PytorchSeq2SeqWrapper(lstm)
+        tensor = torch.rand([5, 7, 3])
+        tensor[0, 3:, :] = 0
+        tensor[1, 4:, :] = 0
+        tensor[2, 0:, :] = 0
+        tensor[3, 6:, :] = 0
+        mask = torch.ones(5, 7)
+        mask[0, 3:] = 0
+        mask[1, 4:] = 0
+        mask[2, 0:] = 0
+        mask[3, 6:] = 0
+
+        initial_states = (Variable(torch.randn(6, 5, 7)),
```

Comment:
Oh, the 6 is because this is bidirectional?  Yeah, a shape comment would have helped a lot here.

Reply:
You could also make one of these tests only use a single variable for the hidden state, instead of two, to make sure we cover that case correctly.